bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -409,4 +409,53 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
+	<!-- Alarm channels -->
+	<channel-type id="alarmSystemState">
+		<item-type>Switch</item-type>
+		<label>System State</label>
+		<description>Alarm system is active (ON) or inactive (OFF) (read only)</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="armedState">
+		<item-type>Switch</item-type>
+		<label>Armed State</label>
+		<description>Alarm is armed (ON) or disarmed (OFF) (read only)</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="alarmNetwork">
+		<item-type>Switch</item-type>
+		<label>Network State</label>
+		<description>Alarm network state (ON = network ok, OFF = no network) (read only)</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="alarmBattery">
+		<item-type>String</item-type>
+		<label>Battery State</label>
+		<description>Alarm battery state (read only)</description>
+		<state>
+			<options>
+				<option value="OK">Ok</option>
+				<option value="FAULT">Fault</option>
+				<option value="UNLOADED">Unloaded</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="zoneAlarm">
```

Comment:
Please update the file openwebnet.properties in i18n folder.
Hint: run `mvn i18n:generate-default-translations`

Reply:
as it was expected the current event options default translations (e.g. `channel-type.openwebnet.cenButtonEvent.event.option.START_PRESS`) get deleted now from the script as it dos not support them yet.
Shall I delete them?

Reply:
I would suggest to apply only the required changes for that PR,

Reply:
i would prefer to remove those commented translations, until event options are supported by the script, otherwise they would be removed each time the script is executed forcing to re-add them manually each time

Reply:
Ok, fine.